### PR TITLE
merge fix to - scientific functions linked to issue

### PR DESCRIPTION
**Pull request by itArnaudov**
tested in lab and confirm okay to merge code with main branch. thx

### CODE REVIEW

**Review by Ivan-CBRE — CHANGES_REQUESTED** [review]
please address the comments section, thanks.

File: PY/calculator/readme.md
[the remark this answers is not in the document]
Comment (itArnaudov) [reply PY/calculator/readme.md:1]: done. thx

File: PY/calculator/calc.py
[the remark this answers is not in the document]
Comment (itArnaudov) [reply PY/calculator/calc.py:1]: completed. thx

**Review by itArnaudov — COMMENTED** [review]
all recommendations are done. thx

**Review by itArnaudov — COMMENTED** [review]
ok

**Review by itArnaudov — COMMENTED** [review]
cool

**Review by itArnaudov — COMMENTED** [review]
approved

**Review by itArnaudov — COMMENTED** [review]
nice one

**Review by itArnaudov — COMMENTED** [review]
good

**Review by itArnaudov — COMMENTED** [review]
approved

**Review by Ivan-CBRE — COMMENTED** [review]
ok

**Review by Ivan-CBRE — COMMENTED** [review]
done

**Review by Ivan-CBRE — COMMENTED** [review]
accept

**Review by Ivan-CBRE — COMMENTED** [review]
good

**Review by Ivan-CBRE — COMMENTED** [review]
read

**Review by Ivan-CBRE — COMMENTED** [review]
good

**Pull request merged**